### DISCUSSION TITLE
Native module RNC_AsyncSQLiteDBStorage tried to override AsyncStorageModule

### DISCUSSION THREAD

**Discussion by ramos-lucas** [react-native-async-storage/async-storage#595] [Q&A]
## Current behavior
I can't run my app on android. This error shows up as soon as the app opens.
I used autolinking

## Expected behavior
Run the application with no issues.

## Repro steps
Install the package e run the app 

## Environment

- **Platforms tested**:
  - [x] Android
  - [ ] iOS
  - [ ] macOS
  - [ ] Windows
- **AsyncStorage version**: 1.15.4
- **Environment**:
```
System:
    OS: macOS 11.2.3
    CPU: (4) x64 Intel(R) Core(TM) i5-5350U CPU @ 1.80GHz
    Memory: 97.06 MB / 8.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 14.13.0 - /usr/local/bin/node
    Yarn: 1.22.5 - /usr/local/bin/yarn
    npm: 6.14.8 - /usr/local/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.10.1 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: iOS 14.3, DriverKit 20.2, macOS 11.1, tvOS 14.3, watchOS 7.2
    Android SDK:
      API Levels: 28, 29, 30
      Build Tools: 28.0.3, 29.0.2, 29.0.3, 30.0.2, 31.0.0
      Android NDK: Not Found
  IDEs:
    Android Studio: 4.1 AI-201.8743.12.41.7042882
    Xcode: 12.3/12C33 - /usr/bin/xcodebuild
  Languages:
    Java: 1.8.0_261 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.1 => 17.0.1 
    react-native: 0.64.0 => 0.64.0 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```
- **Logs/Errors that may be relevant**: 
![image](https://user-images.githubusercontent.com/18932708/116753275-3828da00-a9dd-11eb-88b7-4567c83ff20c.png)

**Comment by tido64** [comment]
It looks like you have two copies of AsyncStorage. Maybe one of the suggested solutions here will help you: https://stackoverflow.com/questions/65351379/native-module-rnc-asyncsqlitedbstorage-tried-to-override-asyncstoragemodule

**Comment by PJVol** [comment]
Checking MainApplication.java file could help, as the error description suggests.
I found three "new AsyncStoragePackage()" there, so as you may guess, removing two of them fixed an issue.
